admin_guide/backup_restore.adoc

```diff
@@ -312,6 +312,11 @@ host:
 ====
 Skip this step if version is lower than `etcd-2.3.7-4` or if etcd database size
 is smaller than 700mb.
+
+If the etcd backup is larger than 700 MB,
+xref:../adming_guide/pruning_resources.html[prune the resource], or clear
```

Comment:
s/adming/admin
s/.html/.adoc  (also remember to include the # and ID (can't be empty)